### PR TITLE
Migrated functions from utils to SDK

### PR DESCRIPTION
**Pull request by mogith-pn**
https://clarifai.atlassian.net/browse/TT-2957

### CODE REVIEW

File: pages/first_page.py [hunk 1]
```diff
@@ -27,19 +27,16 @@
...
 
-  # Stream inputs from the app
-  all_inputs = []
-  for inp in lister.inputs_generator():
-    all_inputs.append(inp)
-
-    if len(all_inputs) >= mtotal:
-      break
+  # Stream inputs from the app. list_inputs give list of dictionaries with inputs and its metadata .
+  input_obj = User(user_id=userDataObject.user_id).app(app_id=userDataObject.app_id).inputs()
```
Comment (sainivedh) [inline pages/first_page.py:31]: As this is a main template page, could we do something different than listing inputs maybe `Image Generation`?!
@sriram-clarifai your inputs please.

Comment (sainivedh) [reply pages/first_page.py:31]: @mogith-pn We should limit all_inputs by `mtotal` variable at here
https://github.com/Clarifai/module-template/blob/9197eb865f0a12547b31130f3721dfc341857569/pages/first_page.py#L36

Comment (sainivedh) [reply pages/first_page.py:31]: @sriram-clarifai If no further comments, we'll proceed with existing `list inputs` functionality in model template page

File: pages/first_page.py [hunk 2]
```diff
@@ -28,19 +28,19 @@
...
-  input_obj = User(user_id=userDataObject.user_id).app(app_id=userDataObject.app_id).inputs()  
-  all_inputs=input_obj.list_inputs()
+  input_obj = User(user_id=userDataObject.user_id).app(app_id=userDataObject.app_id).inputs()
+  all_inputs = input_obj.list_inputs()
 
-#added "data_url" which gives the url of the input.
+  #added "data_url" which gives the url of the input.
   data = []
-  for inp in all_inputs:
+  for inp in range(mtotal - 1):
```
Comment (sainivedh) [inline pages/first_page.py:36]: Any specific reason for `mtotal-1` range by default takes  `0 to mtotal-1`

**Review by sainivedh — COMMENTED** [review]
lgtm

File: pages/first_page.py (hunk 2 above)
Comment (mogith-pn) [reply pages/first_page.py]: There was error showing like index out of range, found the reason which is due to less no of inputs in app. so added exception block and pushed it.

**Review by sainivedh — APPROVED** [review]

**Pull request merged**